src/test/java/api/item/ItemLastCheckInTests.java

```diff
@@ -77,9 +77,10 @@ public void shouldNotFailCheckInWithInvalidLoggedInUserId() {
     UUID servicePointId = servicePointsFixture.cd1().getId();
     DateTime checkInDate = DateTime.now();
 
+    String invalidUuid = "00000000-0000-0000-0000-000000000000";
     final OkapiHeaders okapiHeaders = getOkapiHeadersFromContext()
       .withRequestId("check-in-by-barcode-request")
-      .withUserId("INVALID_UUID");
+      .withUserId(invalidUuid);
```

Comment:
Why did this need to change?

Reply:
I've introduced validation of request against schema for fake item-storage, and `INVALID_UUID` was not a valid value.

Reply:
Does this invalidate the test, as an invalid user ID, that isn't a UUID, will cause this to fail?

Reply:
I think the purpose of this test to verify that checkin won't be failed when there is no user with such ID.

Reply:
My understanding of `invalid` is that the ID provided is in an invalid format, not that the ID does not reference a known record, and hence the user is unknown.

I think that is why an invalid UUID was used, rather than a randomly generated UUID that does not refer to an existing user.

Reply:
In this case, the test case is not relevant. Because item-storage is not allowing such values.

Reply:
Hi @mrdpopov, it seems that this is your test, could you please clarify what is being verified by the case? Are we verifying that check-in works even if the user not found or the user UUID has an invalid format?

Reply:
The purpose of the test is to show that the last checkin does not fail when the supplied user id is invalid.  
I think it is relevant because It shows that passed userID is stored stored in the last checkin object as is(e.g case when userId is not found). I believe, this was in the requirements.

Reply:
Ok, will resolve the comment in this case.